html/topbar.php

```diff
@@ -4,9 +4,9 @@
     <div class="row gx-0 align-items-center" style="height: 45px;">
       <div class="col-lg-6 text-center text-lg-start mb-lg-0">
         <div class="d-flex flex-wrap">
-          <a href="#" class="text-muted me-4"><i class="fas fa-map-marker-alt text-primary me-2"></i>Find A Location</a>
-          <a href="tel:+01234567890" class="text-muted me-4"><i class="fas fa-phone-alt text-primary me-2"></i>+01234567890</a>
-          <a href="mailto:example@gmail.com" class="text-muted me-0"><i class="fas fa-envelope text-primary me-2"></i>Example@gmail.com</a>
+          <a href="#" class="text-muted me-4"><i class="fas fa-map-marker-alt text-primary me-2"></i><?= TOPBAR_LINK ?></a>
+          <a href="tel:+01234567890" class="text-muted me-4"><i class="fas fa-phone-alt text-primary me-2"></i><?= TOPBAR_LINK1 ?></a>
```

Comment:
```suggestion
          <a href="tel:+<?= TOPBAR_TELF ?>" class="text-muted me-4"><i class="fas fa-phone-alt text-primary me-2"></i><?= TOPBAR_LINK1 ?></a>
```
Recuerda crear la constante si no existe